foundations/javascript_basics/fundamentals-2.md

```diff
@@ -1,6 +1,6 @@
 ### Introduction
 
-There are a few extremely common types of data that you will encounter in JavaScript, and these fundamentals lessons will give us a really strong foundation in all of them.  Before we start digging deep, however, [this article](http://javascript.info/types) will give you a quick overview of the most common ones.
```

Comment:
I don't think this is a typo. It's not saying the lesson is fundamental it's saying the lessons cover fundamentals. I suppose it could be reworded to "and these lessons on fundamentals" to make it a bit more precise. I would be open to that change if you want to make it.

Reply:
I agree. If anything, maybe updating the text to, `and these "Fundamentals" lessons`

Reply:
@thatblindgeye That makes more sense to me, even just capitalising the word Fundamentals would be enough I think?